apps/galaxy/src/modules/Orders/OpenOrders/OpenOrdersList.tsx

```diff
@@ -42,16 +67,44 @@ const OpenOrdersList = () => {
     }),
   ];
 
+  const Submit = () => {
+    const Orders = checkedItems
+      .map((checked, index) => {
+        if (checked) return index;
+      })
+      .filter((checked) => checked !== undefined);
+    const orderIds = lists
+      ?.map((checkedOrders: Order, index: number) => {
+        const status = Orders.find((val) => val == index);
+        if (status != undefined) {
+          return checkedOrders.id;
+        }
+      })
+      .filter((value: number | undefined) => value);
+    handleAcceptOrderSubmit(orderIds);
+    (checkedItems.length = lists.length), checkedItems.fill(false, 0, lists.length);
+  };
+
   return (
-    <Stack
-      border="1px"
-      borderBottom="0px"
-      borderColor="gray.200"
-      borderRadius="lg"
-      overflow="hidden"
-    >
-      <DataTable columns={columns} data={lists} />
-    </Stack>
+    <VStack>
+      <HStack>
```

Comment:
Instead of specifying `left` we can leverage css-flex here. 

```tsx
<VStack>
  <HStack alignSelf="flex-end">
    <Button onClick={Submit} isLoading={isLoading} variant="secondary">
      Accept Order(s)
    </Button>
    <Button variant="primary" color="white">
      Reject Order(s)
    </Button>
  </HStack>
</VStack>
```

This is a good guide to keep somewhere to learn the basic of css flexbox.